docs/current/api/975146-concepts.md

```diff
@@ -0,0 +1,126 @@
+---
+slug: /api/concepts
+---
+
+# Key Concepts
+
+## Introduction
+
+One of Dagger's most powerful features is cross-language instrumentation: teams can use each other's tools without learning each other's language. Dagger achieves this by exposing its functionality through GraphQL, which serves as a low-level language-agnostic API framework for all Dagger operations.
+
+This approach has two main benefits:
+
+- It enables CI/CD pipeline authors to use whichever language feels natural and intuitive to them, without having to think about interoperability.
+- Rather than reimplementing the same automation in every possible programming language, it only needs to be implemented once and is then reusable by all. As a result, two cooperating teams no longer necessarily have to come to consensus on a common language.
```

Comment:
This is very well-put, but sadly it will not be true until we ship extensions.

However, it does provide *some* interoperability benefits.

* Common runtime environment. Even if you write pipelines in different languages, they are all run by the same engine. So logging, scaling, monitoring, etc. only need to be solved once for everyone.
* Common low-level language. Even if teams can't directly re-use each other's pipelines (yet) they can take advantage of the fact that they all produce the same GraphQL queries. Similar to how developing for the same platform, even in different languages, creates opportunities for knowledge sharing and collaboration.

Reply:
Added